action/api/api.go

```diff
@@ -8,16 +8,18 @@ import (
 
 type Option func(*action.Config)
 
+//Initiliaize the action.
```

Comment:
Take a look at https://blog.golang.org/godoc-documenting-go-code.  We should probably follow their conventions for the go doc.

It states:

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.
>